include/picongpu/simulation/control/MySimulation.hpp

```diff
@@ -574,18 +577,12 @@ class MySimulation : public SimulationHelper<simDim>
 
     virtual void resetAll(uint32_t currentStep)
     {
-        DataConnector &dc = Environment<>::get().DataConnector();
-
-        auto fieldE = dc.get< FieldE >( FieldE::getName(), true );
-        auto fieldB = dc.get< FieldB >( FieldB::getName(), true );
-
-        fieldB->reset(currentStep);
-        fieldE->reset(currentStep);
-        meta::ForEach< VectorAllSpecies, particles::CallReset< bmpl::_1 > > callReset;
-        callReset( currentStep );
```

Comment:
I am a bit confused by the naming. In the old version you call `callReset` for all species, but now it is called `resetParticles` but I do not see a renaming of this method/function in the new code.

Reply:
The line above defines `resetParticles` and still uses `CallReset` in the template parameters.

Reply:
Ah now I get it. Please excuse my stupidity. @n01r thanks for pointing it out.

Reply:
`callReset` was a variable name defined just 1 line before it is called. With the new version I thought `resetParticles` would be a better name for it.

Reply:
I agree, the naming is better now.

Reply:
Oh, did not see that @n01r already answered.